app/models/support/gds/campaign.rb

```diff
@@ -4,14 +4,29 @@ module Support
   module GDS
     class Campaign
       include ActiveModel::Model
-      attr_accessor :title, :other_dept_or_agency, :signed_campaign, :start_date, :end_date, :description,
-                    :call_to_action, :success_measure, :proposed_url, :site_metadescription, :cost_of_campaign
+      attr_accessor :type_of_site, :has_read_guidance_confirmation, :has_read_oasis_guidance_confirmation,
+                    :signed_campaign, :start_date, :end_date, :development_start_date,
+                    :performance_review_contact_email, :government_theme, :description, :call_to_action,
+                    :proposed_url, :site_metadescription, :cost_of_campaign, :ga_contact_email
 
-      validates_presence_of :title, :signed_campaign, :start_date, :end_date, :description,
-                            :call_to_action, :success_measure, :proposed_url, :site_metadescription, :cost_of_campaign
-      validates_date :start_date, on_or_after: :today, on_or_before: :end_date
-      validates_date :end_date, on_or_after: :start_date
-      validates :proposed_url, format: /((http|https):\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.campaign.gov.uk?/
+      validates_presence_of :type_of_site, :signed_campaign, :start_date, :end_date, :development_start_date,
+                            :performance_review_contact_email, :government_theme, :description, :call_to_action,
+                            :proposed_url, :site_metadescription, :cost_of_campaign, :ga_contact_email
+
+      validates_acceptance_of :has_read_guidance_confirmation, :has_read_oasis_guidance_confirmation, allow_nil: false
+      validates_inclusion_of :type_of_site, in: ['Campaign platform', 'Bespoke microsite']
+
+      validates_acceptance_of :has_read_guidance_confirmation, :has_read_oasis_guidance_confirmation, allow_nil: false
```

Comment:
This line is here twice, I don't think that means it'll have two errors when you forget to check the button, but it might.  Even if rails does the right thing, we should delete one version to avoid confusion when we change one line but not the other.

Reply:
Removed one